tests/MSBuildDeviceIntegration/Tests/BundleToolNoAbiSplitTests.cs

```diff
@@ -0,0 +1,153 @@
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Xamarin.Android.Tasks;
+using Xamarin.ProjectTools;
+using Xamarin.Tools.Zip;
+
+namespace Xamarin.Android.Build.Tests
+{
+	[TestFixture]
+	[Category ("XamarinBuildDownload")]
```

Comment:
not sure we need this category? Maybe it should b "UsesDevice"

Reply:
I just copied it from the other bundle test case (`BundleToolTests`)